src/constants/config.py

```diff
@@ -12,11 +12,14 @@
     DB_PORT=os.getenv("DB_PORT"),
     DB_NAME=os.getenv("DB_NAME"),
     DB_SSL_MODE=os.getenv("DB_SSL_MODE") or "require",
-    # Database retry/timeout constants
+    # Database retry/timeout/keepalive
     DB_CONN_RETRY_LIMIT=8,
     DB_CONN_SLEEP_START=5,
     DB_CONN_SLEEP_MAX=60,
     DB_CONN_TIMEOUT=5,
+    DB_KEEPALIVE_IDLE=os.getenv("DB_KEEPALIVE_IDLE"),
```

Comment:
Could there be defaults here or in .env-example? Make it easier to set up local environments

Reply:
Will add in `.env-example`